src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,41 @@
+package bit.javaoop;
+
+public class FixedBudgetManager extends AbstractManager {
+    private final String position ="manager";
+
+    @Override
+    public String getPosition() {
+        return position;
+    }
+
+    public FixedBudgetManager(String name, Float salary)
+    {
+        super(name,salary);
+
+        getEmployeeList().add(this);
```

Comment:
I think you shouldn't do this. This `employeeList` should contain only manager's employees - his or hers subordinates.